aip/general/0200.md

```diff
@@ -13,7 +13,7 @@ redirect_from: /not-precedent
 Many times, APIs are written in ways that do not match new guidance that is
 added to these standards after those APIs have already been released.
 Additionally, sometimes it can make sense to intentionally violate standards
-for particular reasons, such as maintaining consistency with established
```

Comment:
I'm fine with the others, but imho, this is one place where "consistency" is better.

Reply:
sounds good! I was on the fence here anyway.